client/src/modules/Teams/components/TeamModal/TeamModal.tsx

```diff
@@ -0,0 +1,93 @@
+import { Form, Input, Space, Typography } from 'antd';
+import TextArea from 'antd/lib/input/TextArea';
+import Modal from 'antd/lib/modal/Modal';
+import { CreateTeamDto, TeamDto } from 'api';
+import { urlPattern } from 'services/validators';
+
+type Props = {
+  data: Partial<TeamDto>;
+  onCancel: () => void;
+  onSubmit: (record: CreateTeamDto, id?: number) => Promise<void>;
+};
+
+const { Text } = Typography;
+
+const layout = {
+  labelCol: { span: 6 },
+  wrapperCol: { span: 20 },
+};
+
+export default function TeamModal({ onCancel, onSubmit, data }: Props) {
+  const [form] = Form.useForm<Partial<TeamDto>>();
+
+  const createRecord = (values: Partial<TeamDto>): CreateTeamDto => {
+    return {
+      name: values.name!,
+      description: values.description!,
+      chatLink: values.chatLink!,
```

Comment:
it is better set default values rather than using `Partial` and then using non null assertion

Reply:
fixed